src/components/header.css

```diff
@@ -0,0 +1,84 @@
+.header {
+    box-shadow: 0px 1px 5px rgba(0, 0, 0, 0.1);
+    background-color: #fff;
+    min-height: 50px;
+    display: flex;
+    flex-direction: column;
+    position: fixed;
+    width: 100vw;
+    z-index: 10;
+    top: 0;
+    /* padding: 5px; */
+    justify-content: center;
+  }
+  
+  .navigation {
+    align-items: center;
+    display: flex;
+    padding: 0 40px 0 20px;
+    position: relative;
+  }
+  
+.horizontal-menu {
+    list-style: none;
+    display: block;
+    flex-direction: row;
+    justify-content: flex-start;
+    flex-wrap: nowrap;
+    align-items: center;
+    margin: 0 10px;
+    padding: 0;
+    font-size: 0.7em;
+    overflow-x: auto;
+    width: 100%;
+    white-space: nowrap;
+  }
+
+  ::-webkit-scrollbar { 
```

Comment:
shouldn't that be attached to some selector, so only specific embedded scrollbar is disabled rather than global ?

Reply:
done